src/components/CreateTarget/CreateTarget.test.js

```diff
@@ -0,0 +1,157 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import { default as userEvent } from '@testing-library/user-event';
+import { BrowserRouter } from 'react-router-dom';
+
+import { MapContext } from 'contexts/MapContext';
+import CreateTarget from './CreateTarget';
+
+const mapProperties = {
+  selectedRadius: 100,
+  selectedLocation: { lat: 10, lng: 10 },
+};
+
+const values = {
+  mapProperties,
+  setMapProperties: jest.fn(),
+  targets: [],
+  setTargets: jest.fn(),
+};
+
+const mockGoTo = jest.fn();
+
+const useContentView = () => ({
+  displayedComponent: 'Home',
+  goTo: mockGoTo,
+  isMapVisible: false,
+  setIsMapVisible: jest.fn(),
+  navbarColor: 'blue',
+  setNavbarColor: jest.fn(),
+  navbarLeftButton: 'back',
+  setNavbarLeftButton: jest.fn(),
+});
+
+jest.mock('hooks/useContentView', () => useContentView);
+
+const CreateTargetHarness = ({ values }) => (
+  <MapContext.Provider value={values}>
```

Comment:
you could have a name conflict here, at line 13 you define a const `values` too

Reply:
if you always pass the same values, you can omit to receive it by params
```suggestion
const CreateTargetHarness = () => (
  <MapContext.Provider value={values}>
```